static/app/views/profiling/profileDetails.tsx

```diff
@@ -164,22 +284,63 @@ function ProfileDetails() {
                 placeholder={t('Search for frames')}
                 onChange={handleSearch}
               />
+
+              <CompactSelect
+                options={columnFilters.type.values.map(value => ({value, label: value}))}
+                value={filters.type}
+                triggerLabel={
+                  !filters.type ||
```

Comment:
yeah, i can in some commentary. 

from a UX perspective, i would find it weird to have all _n_ values checked, and then a user having to go uncheck a large number of filters just to selectively check some.

the gist here is, we render "All" if theres no filter applied. if a user checks every possible value, we also render "All".

i opted to go about this way since we don't have a nice "select all" or "clear" option given the current constraints of our ui library.